coalib/results/Diff.py

```diff
@@ -358,7 +358,7 @@ def delete_lines(self, line_nr_start, line_nr_end):
         """
         Delete lines in a specified range.
         """
-        for line_nr in range(line_nr_start, line_nr_end):
+        for line_nr in range(line_nr_start, line_nr_end + 1):
```

Comment:
@Makman2 - Should I just be updating the comments or is there documentation somewhere specific that should be updated?